src/app/faq-block/faq-block.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="faqblock">
+  <h2>{{headline}}</h2>
+  <div class="questionsAnswers">
+    <div *ngFor="let item of faq;let index=index;">
+      <div class="collapsible-toolbar">
+        <span class="collapsible-toolbar__title question">{{item.question}}</span>
+        <span class="collapsible-toolbar__toggle" (click)="toggle[index] = !toggle[index]">{{toggle[index] ? 'ausblenden' : 'einblenden'}}</span>
+        <i class="far" [ngClass]="toggle[index] ? 'fa-chevron-down' : 'fa-chevron-right'"></i>
+      </div>
+      <div *ngIf="toggle[index]" id="faqAnswer{{index+1}}" class="answer" [innerHTML]="item.answer"></div>
+    </div>
```

Comment:
I would separate this into an own `FaqItemComponent`.
That way you could have a `isToggled` property for each individual item, instead of using this (for me) confusing array of booleans.

Reply:
Done